google/cloud/storage/tests/object_basic_crud_integration_test.cc

```diff
@@ -36,8 +37,27 @@ namespace {
 using ::testing::Contains;
 using ::testing::Not;
 using ::testing::UnorderedElementsAreArray;
-using ObjectBasicCRUDIntegrationTest =
-    ::google::cloud::storage::testing::ObjectIntegrationTest;
+
+struct ObjectBasicCRUDIntegrationTest
+    : public ::google::cloud::storage::testing::ObjectIntegrationTest {
+ public:
+  static Client MakeNonDefaultClient() {
```

Comment:
this isn't obviously the same, but I will bet that you know what you are doing without digging deeper.

Reply:
Ack. Thanks.  I reviewed again just to make sure.